compiler+runtime/test/cpp/jank/runtime/obj/ratio.cpp

```diff
@@ -0,0 +1,401 @@
+#include "jank/runtime/obj/ratio.hpp"
+#include <doctest/doctest.h>
+#include <stdexcept>
+
+namespace jank::runtime
+{
+  TEST_SUITE("ratio_data")
```

Comment:
Let's keep all of this in one `ratio` test suite. That'll make it easier to run from the cli, using `-ts=ratio`. You can separate the ratio_data tests with test cases or sub cases, if you'd like.

Reply:
updated.